doc/source/contributing/write_examples.rst

```diff
@@ -10,21 +10,19 @@ If you want to translate a MAPDL script into a PyMAPDL one, you can first
 run the following commands:: 
 
     from ansys.mapdl.core import convert_script
-
     convert_script(path_MAPDL_script)
 
 .. warning::
 
-    The ``mapdl.convert_script`` function is under development and
-    needs to be improved.
+    The ``mapdl.convert_script`` function is still on beta state, hence it is advised you check its output.
     Please, double-check if there are easier ways to write some commands.
```

Comment:
```suggestion
    Because the ``mapdl.convert_script`` method is still in beta, you should check its output.
    Always determine if there are easier ways to write any of the commands.
```